examples/angular/README.md

```diff
@@ -0,0 +1,29 @@
+# Angular Example
+
+This is a demo of the usage of ArmoniK Api for Angular.
+
+Please, refer to the documentation to get more information about the usage of the library. We write a special guide to help you to use ArmoniK Api with Angular.
+
+## How to run
+
+_You must have an ArmoniK instance running to run this example._
+
+To run this example, you need to install the dependencies first:
+
+```bash
+pnpm install
+```
+
+Then, you must create a `src/proxy.conf.json` from the `src/proxy.conf.json.example` file. You can do it by running:
```

Comment:
Then, Create a `src/proxy.conf.json` from the `src/proxy.conf.json.example` file. To do so, run: